### DISCUSSION TITLE
Bug: drawing canvas blurry when canvas upside

### DISCUSSION THREAD

**Discussion by Lioncub901** [keiwando/toonsquid#89] [Q&A] [answered]
I am making a pixel animation just for fun with a canvas size of 60 by 60. The problem is when I rotate the canvas (flip it upside down) the canvas is no longer pixelated but becomes blurry. It does return to it pixelated state when turn the canvas the right way up.

Normal: 

![IMG_0920](https://user-images.githubusercontent.com/88890833/205460998-47828c60-96ba-4eb9-a242-69f0b93b8945.png)

Upside down:

![IMG_0922](https://user-images.githubusercontent.com/88890833/205461049-17e9007a-ec39-480d-9b10-64d582ec87ce.png)

**Comment by keiwando** [comment]
Thanks for reporting this bug! I will look into it.

**Comment by keiwando** [comment, marked as answer]
This should now be fixed with 1.7.1
